src/PrestaShopBundle/Resources/config/routing/admin/sell/catalog/categories.yml

```diff
@@ -12,16 +12,16 @@ admin_categories_index:
         categoryId: '\d+'
 
 admin_categories_search:
-    path: /search
-    methods: [POST]
-    defaults:
-        _controller: PrestaShopBundle:Admin/Sell/Catalog/Category:searchGrid
-        _legacy_controller: AdminCategories
-        _legacy_link: AdminCategories:submitFiltercategory
-        gridDefinitionFactoryServiceId: prestashop.core.grid.definition.factory.category
-        redirectRoute: admin_categories_index
-        redirectQueryParamsToKeep:
-            - 'categoryId'
+  path: /
```

Comment:
Changing the path might be a BC break (although I'm in favor of it)

Reply:
Fixed